src/examples/form/nested-form.tsx

```diff
@@ -0,0 +1,78 @@
+import { Component, h, State } from '@stencil/core';
+
+const schema = {
+    type: 'object',
+    properties: {
+        name: {
+            type: 'string',
+            title: 'Name',
+        },
+        age: {
+            type: 'integer',
+            title: 'Age',
+        },
+        address: {
+            type: 'object',
+            title: 'Location',
+            description: 'Please enter your location',
+            properties: {
+                city: {
+                    type: 'string',
+                    title: 'City',
+                },
+                country: {
+                    type: 'string',
+                    title: 'Country',
+                    description: 'It might also be a planet',
+                },
+            },
+        },
+        data: {
+            type: 'object',
+            collapsible: true,
+            title: 'Data',
+            description: 'Some additional data we would like you to submit',
```

Comment:
I added this line as well, so there's a description in a collapsible-section in the example.